internal/website/content/ref/sql.md

```diff
@@ -25,14 +25,15 @@ https://godoc.org/gocloud.dev/postgres.
 
 For MySQL:
 
-* [GCP Cloud SQL](https://godoc.org/gocloud.dev/mysql/cloudmysql)
-* [AWS RDS](https://godoc.org/gocloud.dev/mysql/rdsmysql)
+* [GCP Cloud SQL](https://godoc.org/gocloud.dev/mysql/gcpmysql)
+* [AWS RDS](https://godoc.org/gocloud.dev/mysql/awsmysql)
```

Comment:
Same as above:
AWS RDS for MySQL
AWS RDS MySQL
AWS MySQL
(https://www.google.com/search?q=aws+mysql)

Reply:
Done.